src/theme/scss/components/_quotes-component.scss

```diff
@@ -6,41 +6,83 @@
 
   @extend .global-padding-left-right;
 
-  &--background-shape-a55{
-    width: 140px;
-    margin-bottom: 48px;
+  &--card {
+    background-color: var(--color-background-borrowers-quote);
+    width: 720px;
+    height: 420px;
+    margin-top: 64px;
+    margin-right: 96px;
```

Comment:
add padding of 100px and remove margin/padding-left for childs in the card, let the parent determine the content